src/passkit.cs

```diff
@@ -1554,15 +1605,19 @@ interface PKPaymentErrorKeys {
 
 	interface IPKDisbursementAuthorizationControllerDelegate { }
 
+#if !XAMCORE_5_0
 	[NoMac] // only used in non-macOS API
 	[NoWatch]
 	[iOS (12, 2)]
 	[MacCatalyst (13, 1)]
+	[Obsoleted (PlatformName.iOS, 17, 0, message: "No longer used.")]
+	[Obsoleted (PlatformName.MacCatalyst, 17, 0, message: "No longer used.")]
 	[Native]
 	public enum PKDisbursementRequestSchedule : long {
 		OneTime,
 		Future,
 	}
+#endif
```

Comment:
Looks like the instance of this enum being used was removed and moved to PKCompat:

```cs
#if NET
		[UnsupportedOSPlatform ("macos")]
		[UnsupportedOSPlatform ("ios")]
		[UnsupportedOSPlatform ("maccatalyst")]
#endif // NET
		[EditorBrowsable (EditorBrowsableState.Never)]
		[Obsolete ("This property is removed.")]
		public virtual PKDisbursementRequestSchedule RequestSchedule {
			get => throw new InvalidOperationException (Constants.RemovedFromPassKit);
			set => throw new InvalidOperationException (Constants.RemovedFromPassKit);
		}
```

 Is this the correct way to remove this enum?

Reply:
Yes, that's correct!